molecule/pdmysql/pdps-minor-upgrade/tasks/main.yml

```diff
@@ -117,10 +120,12 @@
     with_items:
     - rm -rf /package-testing
     - rm -f master.zip
-    - wget --no-check-certificate -O master.zip https://github.com/Percona-QA/package-testing/archive/master.zip
+    - wget --no-check-certificate -O master.zip "https://github.com/Percona-QA/package-testing/archive/{{ branch }}.zip"
```

Comment:
Why we are downloading "package-testing" inside the package-testing test? It's already here and it's already on required branch.

Reply:
https://www.screencast.com/t/lryqrrHvT5A1